package.json

```diff
@@ -92,6 +93,7 @@
     "@types/jasmine": "~2.8.0",
     "@types/jasminewd2": "^2.0.3",
     "@types/node": "^10.11.0",
+    "@types/pegjs": "^0.10.2",
```

Comment:
Any reason why this version is not the same as the `pegjs` version above?

Reply:
@adnanakgun, are you able to comment on this one?

Reply:
I am using a tool that imports that stuff automatically. I'll try to match versions and see if that breaks anything. Good catch.

Reply:
Made the change @paul-graham .